graphql/resolvers.go

```diff
@@ -1417,6 +1417,26 @@ func (r *mutationResolver) RemovePublicKey(ctx context.Context, keyName string)
 	return myPublicKeys, nil
 }
 
+func (r *mutationResolver) UpdatePublicKey(ctx context.Context, targetKeyName string, updateInfo PublicKeyInput) ([]*restModel.APIPubKey, error) {
+	if !doesPublicKeyNameAlreadyExist(ctx, targetKeyName) {
```

Comment:
I'm a bit confused by what this means for the request format. What's the reason for having 2 different request fields for the key name? Would it be sufficient to only have 1?

Reply:
The Edit Public Key modal allows you to adjust the key name along with the key value. That's why included both fields in `updateInfo`. 
https://mongodb.invisionapp.com/d/main#/console/19647524/426880067/preview